### PR TITLE
:memo: MIGRATING.md

### PR DESCRIPTION
**Pull request by fabienjuif**
fixes #135

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### State
+The state shape used by `k-redux-factory` is changed to be less bloated when serialized.
+Under the hood we use Javascript *Map* object.
+
+This is a breaking change when you:
+ - manually retrieve the substate from a `k-redux-factory` reducer
+ - or use the `getState` selector
+
+All other selectors are impacted and should work as before 👌
```
Comment (delphinemillet) [inline MIGRATING.md]: are NOT ?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,9 @@ Factory of Redux reducers and their associated actions and selectors.
 [![CircleCI](https://circleci.com/gh/alakarteio/k-redux-factory.svg?style=shield)](https://circleci.com/gh/alakarteio/k-redux-factory) [![Coverage Status](https://coveralls.io/repos/github/alakarteio/k-redux-factory/badge.svg?branch=master)](https://coveralls.io/github/alakarteio/k-redux-factory?branch=master) [![NPM Version](https://badge.fury.io/js/k-redux-factory.svg)](https://www.npmjs.com/package/k-redux-factory)
 [![Size](http://img.badgesize.io/alakarteio/k-redux-factory/master/dist/index.js.svg)]() [![Greenkeeper badge](https://badges.greenkeeper.io/alakarteio/k-redux-factory.svg)](https://greenkeeper.io/)
 
+## Migrating
+Hey ! If you come from an early version of `k-redux-factory` and want to upgrade, [you can read this migrating guide 💎](./MIGRATING.md)
```
Comment (delphinemillet) [inline README.md]: migration guide ?

File: MIGRATING.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ - `orderBy` action is removed
+    * You can wrote [your own with a middleware](./README.md#example-we-create-a-middleware-but-we-modify-action-and-state-)
+    * You can use [k-ramel](https://github.com/alakarteio/k-ramel) to handle your logical code and use `k-redux-factory` for the key/value store only
+    * You can use [redux-saga](https://github.com/redux-saga/redux-saga) to handle your logical code and use `k-redux-factory` for the key/value store only
+ - `factory` export is not the default one
+    * **Before**: `import factory, { keyValue } from 'k-redux-factory'`
+    * **Now**: `import { factory, keyValue } from 'k-redux-factory'`
+ - `state` created by `k-redux-factory` is modified to be less bloated when serialized:
+    * `getState` selector returns a new shape, see [dedicated chapter](#state)
+    * all other selectors are impacted and should work as before 👌
```
[the remark this answers is not in the document]
Comment (fabienjuif) [reply MIGRATING.md]: - I have impacted them so it works as before
 - The user should NOT be impacted

[the remark this answers is not in the document]
Comment (delphinemillet) [reply MIGRATING.md]: Usually "impacted" and "work as before" are not applied at the same time to describe something.
Furthermore, to me the word "impacted" makes me a bit anxious.

but ok !

**Pull request merged**